pkg/common/config/config.go

```diff
@@ -77,7 +77,7 @@ const (
 	CanaryChance = "canaryChance"
 
 	// Default network provider for OSD
-	DefaultNetworkProvider = "OpenShiftSDN"
+	DefaultNetworkProvider = "OVNKubernetes"
```

Comment:
This const is used in `pkg/common/providers/rosaprovider/cluster.go` only?
I think we can remove that part as well.